nrrd-design-system/components/components/selectors/selectors.config.yml

```diff
@@ -1,5 +1,21 @@
 title: Selectors
 status: blocked
```

Comment:
This wasn't in the completion criteria before, but I realized that with these fixes in, we can change this status from `blocked` to `draft`. That will also start the Circle CI build again, which was having issues last week, but is performing better this week and should pass now.